src/main/java/org/folio/fqm/migration/strategies/V0POCMigration.java

```diff
@@ -442,14 +442,14 @@ public Map<UUID, Map<String, BiFunction<String, String, FieldWarning>>> getField
         OLD_DRV_LOAN_DETAILS,
         Map.of(
           "instance_primary_contributor",
-          (String field, String fql) -> new QueryBreakingWarning(field, "instance.contributors", fql)
+          (String field, String fql) -> new QueryBreakingWarning(field, "instance.instance_primary_contributor", fql)
         )
       ),
       Map.entry(
         OLD_DRV_ITEM_DETAILS,
         Map.of(
           "instance_primary_contributor",
-          (String field, String fql) -> new QueryBreakingWarning(field, "instance.contributors", fql)
```

Comment:
Since the primary field has returned, you can entirely remove this Warning, and update the map higher up in this file